<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpiritThumb</class>
 <widget class="QWidget" name="SpiritThumb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="SpiritId">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 10pt &quot;Press Start 2P&quot;;</string>
   </property>
   <property name="text">
    <string>No.00000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="SpiritImg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(192, 192, 192);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
